PWGGA/GammaConv/macros/AddTask_GammaCalo_pp.C

```diff
@@ -1218,6 +1218,7 @@ void AddTask_GammaCalo_pp(  Int_t     trainConfig                   = 1,
    // ************************************* DCal cuts ****************************************************
   } else if (trainConfig == 601){ // DCAL clusters pp 5.02 TeV
     cuts.AddCut("00010113","3115511081041220000","0163103100000050"); // std
+    cuts.AddCut("00055113","3115511081041220000","0163103100000050"); // std
```

Comment:
Could you also add the triggers for DG2 and DG1 as the DMC7 was firing together with the MB only.